Jenkinsfile

```diff
@@ -0,0 +1,91 @@
+G_gitcred = 'TonJenSSH'
+G_container = "atomxy/empty-ton-sdk-js:20191128"
```

Comment:
Could we use official `rust:1.39` here instead of custom image? It's more predictable and does not need any support from our side.
Tests are passed with this change: https://builder.tonlabs.io/jenkins/job/Node/job/ton-labs-types/job/CI/2/replay/diff

Reply:
Implemented